apps/external-devices/device-descriptors/PhaseSpace_Motion_Capture.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deviceVendor": "PhaseSpace",
+    "deviceName": "Motion Capture",
+    "author": "Kevin M. Godby <kevin@godby.org>",
+    "version": 1,
+    "lastModified": "2016-12-06T22:17:44Z",
+    "interfaces": {
+        "analog": {
```

Comment:
I think the `0`s trigger a validation failure, too.. I just haven't set up the validator on this computer yet.